README.md

```diff
@@ -47,9 +26,23 @@ Everyone must sign the Presto CLA prior to making a contribution. You may either
 
 ## Members
 
-The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Tim Meehan ([@tdcmeehan](https://github.com/tdcmeehan)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
+The current members of the Presto TSC are:
+
+| Name | Github | Affiliation | Term begins | Term ends |
+| ---- | ------ | ------------|-------------|-----------|
+| Bin Fan  | apc99 | November 1, 2021 | TBD | TBD |
```

Comment:
Affiliation is aligned to the values of `term begins`. Do we wanna put the actual companies?

Reply:
You're right, I copied the wrong header. One second and I'll push a revision.

As long as the charter contains the provision that restricts votes to one per company in §3.a, it would make sense to track this info. This column can be removed if/when that provision is removed.